src/content/guidelines/color/overview.md

```diff
@@ -0,0 +1,227 @@
+        <img src="images/High_contrast_02.png" alt="High contrast example in a dark theme.">    
+    </do-dont-example>
+</grid-wrapper>
+
+
+## Interaction states
+
+In addition to the enabled state tokens above, the five other primary interaction states have also been defined and tokenized for each theme. For values between Black and 70, interaction gets lighter and for values between 60 and White, interaction gets darker.
+
+#### Hover
+A unique set of hex values and color tokens exists for the hover states. These colors are outside of the core palette steps and can be considered as half steps in between two adjacent colors. In order to simplify our token structure, the White theme shares the same hover value as the Gray 10 theme; the two dark theme backgrounds share a hover value as well.
+
+#### Active
+The active state is two full steps lighter for values between 100 and 70 and two full steps darker for values between 60 and 10. Exception: White shares the same actived state as Gray 10 and Black shares the same active state as Gray 100. Example: the Blue 60 active state is Blue 80.
+
+#### Selected
+The selected state is one full step lighter for values between 100 and 70 and one full step darker for values between 60 and 10. Exception: White shares the same selected state as Gray 10 and Black shares the same selected state as Gray 100. Example: the Blue 60 selected state is Blue 70.
+
+#### Focus
+The focus state usually appears as a Blue 60 border in the light theme and as a White border in the dark theme.
+
+#### Disabled
+Disabled components are unavailable for interaction, so they don't receive hover or focus and are not subject to WC3 contrast compliance standards. For the dark themes, disabled values are two full steps lighter than their respective background. For light themes, two full steps darker. Exception: The White theme disabled value is Gray 30.     
+
+
+
+
+![Interactive states light theme](images/Interactive_states_light_new.png)
+
+_Light theme interactive tokens_
+
+
+![Interactive states dark theme](images/Interactive_states_dark_new.png)
+
+_Dark theme interactive tokens_
+
+
+## Digital Accessibility
+
+
+Using various forms of contrast is the most important consideration when making user-friendly color and interface choices. Awareness of standards and color blindness best practices is the key to accessible color selection.
+
+### WCAG 2.0 standards
+
+All type color combinations must pass WCAG AA standards of 4.5:1 for normal text and 3:1 for large text. For larger text, if the font weight is light (300) or normal (400) the text should be no smaller than 24px. If the font weight is Semi-Bold (600) then the large text should be no smaller than 19px.
+
+The color families in our palette contain ten values from 10 to 100. White and Black sit outside those values. Black text is WCAG AA accessible on colors ranging from 10 to 50. White text is accessible on colors from 60 to 100.
```

Comment:
"The color families in ~our~ **the IBM Design Language** palette..."
maybe with a hyperlink as well?